src/android.js

```diff
@@ -0,0 +1,18 @@
+import Router from 'koa-router';
+import moment from 'moment';
+import { getSpreadsheetValues, androidSpreadsheetId } from './utilties';
+
+export const router = new Router();
+
+router.get('/klar', async (ctx) => {
+  const { site } = ctx.request.query;
+  const list = await getSpreadsheetValues({
+    id: androidSpreadsheetId,
+    range: `${site}`,
```

Comment:
Do you need to quote site? Wouldn't `range: site,` do the trick?

Reply:
hah, yes - missed that!